### DISCUSSION TITLE
Blocked myself out from cosmos due to geoblock, still have ssh access to server

### DISCUSSION THREAD

**Discussion by kun-codes** [azukaar/Cosmos-Server#341] [Q&A]
Hi, I blocked myself out from my cosmos install after I whitelisted only India from the geoblock. Although I am accessing it from India, it isn't working. Can you tell me if there is a way to reset this geoblock using ssh access?

**Comment by azukaar** [comment]
Go to /var/lib/cosmos/cosmos.config.json and search for countries

**Comment by kun-codes** [comment]
![image](https://github.com/user-attachments/assets/02cedd70-1beb-4f74-a1e3-fe5654f43406)

Are you talking about this? Also why is this blocking me out when I am trying to access it from India? My cosmos server should be only accessible from India right?

**Reply by Jamblond** [reply]
[**According to Wikipedia**](https://en.wikipedia.org/wiki/IP_address_blocking#How_it_works) (in simple words) webistes can guess which country you came from because every IP address are linked to an [ISP](https://en.wikipedia.org/wiki/Internet_service_provider) and each ISP are "linked" to a country. Or at least this is how I understand it 🥲 
And you may be blocked because you are using a VPN or a Proxy.
To "unlock" your Cosmos access you just need to remove {**"IN"**} from this file (based on your screenshot).

PS: I have limited knowledge about Internet networks, sorry if I made a mistake
And sorry for bad English

**Reply by kun-codes** [reply]
yes I was able to do it. Thanks

**Comment by Jamblond** [comment]
@kun-codes Is your issue fixed now?

**Reply by kun-codes** [reply]
yes it is
